tscreen.go

```diff
@@ -139,7 +139,23 @@ func (t *tScreen) Init() error {
 
 	if t.ti.SetFgBgRGB != "" || t.ti.SetFgRGB != "" || t.ti.SetBgRGB != "" {
 		t.truecolor = true
+	} else if terminalSupportsTrueColor() {
+		// TODO: share with the terminfo package?
+		const (
+			addSetFgRGB = "38;2;%p1%d;%p2%d;%p3%d"
+			addSetBgRGB = "48;2;%p1%d;%p2%d;%p3%d"
+		)
+		t.ti.SetFgRGB = "\x1b[" + addSetFgRGB + "m"
+		t.ti.SetBgRGB = "\x1b[" + addSetBgRGB + "m"
+		// NOTE combined fg/bg setting doesn't work in tmux at least TODO
+		// research more context, this may be why (neo)vim documents so
+		// strongly `It uses the "setrgbf" and "setrgbb"` (saying nothing of
+		// combined fg/bg setting).
+		// t.ti.SetFgBgRGB = "\x1b[" + addSetFgRGB + ";" + addSetBgRGB + "m"
```

Comment:
This deserves a little more research, since not using the combined form represents significantly more data being sent to the terminal (and thus a significant performance penalty.)

Reply:
I'd like to confirm this -- maybe we need an override for tmux here?  I'm fairly opposed to *not* using the combined capability for any terminal that supports it (its a big performance win -- especially e.g. for programs that frequently update large areas of the screen like games).